dist/styles.css

```diff
@@ -1,3 +1,7 @@
 * {
   box-sizing: border-box;
 }
+
+textarea {
+  width:
+}
```

Comment:
🤔

Reply:
hah!  totally forgot to remove this when i learned that the cols and rows attributes are required for textarea elements.  thanks tim.